tests/ping.go

```diff
@@ -101,3 +102,28 @@ func vmiConsoleExpectBatch(vmi *v1.VirtualMachineInstance, expected []expect.Bat
 	}
 	return err
 }
+
+// PingAppFromVirtHandler performs a natcat tcp check through the provided virt-handler pod.
```

Comment:
Using virt-handler here feels like an abuse. Using the deployed application as a utility for testing, expecting it has nc available. It may cause issues on D/S in case virt-handler is built from a different base image that does not have this binary.

Could you spawn a VM for this which would be eventually replaced by a dedicated Pod? The new container image for testing should be added to https://github.com/kubevirt/kubevirt/tree/master/cmd but it may take quite some time to get it right.

Reply:
virt-handler is used by the tests for much more than this, e.g. the reporting and accessing the node.
I think depending on it in other places seems fair enough.

I see no need to spawn a VM  from such a simple task.
I'm fine to replace it in the future with a dedicated pod/job that can be used for such tasks.

> It may cause issues on D/S in case virt-handler is built from a different base image that does not have this binary.

Originally I wanted to reach the node and do that from there, but it is the node that has different binaries (in this case the kind provider did not have it).
If we do not see a problem now, I think it is a good enough compromise to use it.

Reply:
It must not use virt-handler. nc is not our dependency as is in the image only coincidentally. I can speak for our D/S where we use a minimal base image for virt-handler and nc is not available there. Please pick a different solution so these tests can be executed on an arbitrary cluster.